css/src/core/animations.scss

```diff
@@ -24,3 +24,18 @@
 		transform: scale(1);
 	}
 }
+
+// stylelint-disable-next-line keyframes-name-pattern
```

Comment:
do we really need to disable lint here? seems like we could run rename the animation, same for below

Reply:
Fixed 👍🏻